tests/exceptions/exceptions_make_error_code.cpp

```diff
@@ -0,0 +1,118 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for sycl::make_error_code function
+//
+*******************************************************************************/
+
+#include "exceptions.h"
+
+#define TEST_NAME exceptions_make_error_code
+
+namespace TEST_NAMESPACE {
+
+using namespace sycl_cts;
+
+/** @brief Provide verification for sycl::make_error_code function
+ *  @param err_c Error code from sycl::errc enumeration
+ *  @param log sycl_cts::util::logger class object
+ */
+void check_sycl_working(sycl::errc err_c, util::logger &log) {
+  std::error_code make_err_c_result{sycl::make_error_code(err_c)};
```

Comment:
```suggestion
  auto make_err_c_result{sycl::make_error_code(err_c)};
```
This way `make_err_c_result` type matches `sycl::make_error_code(err_c)` return type and `CHECK_TYPE` below does relevant check.

Reply:
Done.